.github/workflows/ssl_test.yml

```diff
@@ -0,0 +1,89 @@
+name: Check ssl functionality
+
+on:
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  test_ssl:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Cache Composer dependencies for improve speed
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/composer
+        key: ${{ runner.os }}-drupal
+
+    - name: Get Drupal Project repository
+      uses: actions/checkout@v2
+      with:
+        repository: 'drupal/recommended-project'
+        path: './'
+        ref: '9.2.x'
+
+    - name: Run Composer install
+      run: composer install --ignore-platform-reqs
+
+    - name: Get Drupal Dockerizer repository
+      uses: actions/checkout@v2
+      with:
+        path: '.drupal-dockerizer'
+
+    - name: Create basic dockerizer.yml
+      run: |
+        echo "---" > dockerizer.yml
+        echo "" >> dockerizer.yml
+        echo "compose_project_name: drupal-project" >> dockerizer.yml
+        echo "user_uid: $(id -u)" >> dockerizer.yml
+        echo "user_gid: $(id -g)" >> dockerizer.yml
+        echo "drupal_root_dir: $GITHUB_WORKSPACE" >> dockerizer.yml
+        echo "advanced_networking: true" >> dockerizer.yml
+        echo "network_id: 15" >> dockerizer.yml
+        echo "domain_name: drupal.devel" >> dockerizer.yml
+
+    - name: Run main playbook
+      uses: dawidd6/action-ansible-playbook@v2
+      with:
+        playbook: main.yml
+        directory: .drupal-dockerizer
+        options: |
+          --verbose
+
+    - name: Run drush-commands playbook
+      uses: dawidd6/action-ansible-playbook@v2
+      with:
+        playbook: drush-commands.yml
+        directory: .drupal-dockerizer
+        options: |
+          --verbose
+
+    - name: Check drupal.devel domain work
+      run: curl http://drupal.devel | grep "Drupal"
+
+    - name: Generate certificates for next step
+      uses: kofemann/action-create-certificate@v0.0.4
+      with:
+        hostcert: 'cert.pem'
+        hostkey:  'private.key'
+        cachain:  'ca-chain.pem'
```

Comment:
It's redundant. Let's remove it.

Reply:
It`s required option for this action https://github.com/kofemann/action-create-certificate/blob/master/action.yml#L18